Yoast/wordpress-seo
# Hide save button when there's nothing to save

## Summary

This PR can be summarized in the following changelog entry:

* Hide the "Save changes" button on tabs where you can't actually save anything.

## Test instructions

This PR can be tested by following these steps:

* Go to Dashboard → General, see that the Save changes button is gone.

## Quality assurance


Fixes #6569
